source/whats-new.txt

```diff
@@ -35,6 +35,28 @@ The 4.8 driver ends support for MongoDB v3.4 and earlier. To learn
 more about breaking changes in this driver version and how to address 
 them, see :ref:`<java-server-release-change-v4.8>`.
 
+Behavioral changes with the 4.8 driver release include:
+
+- Specification of explicit dependency on the ``org.bson.codecs.record``
+  module if your application deploys the driver in an OSGi container and
+  relies on the driver for encoding and decoding Java records.
```

Comment:
Issue: The way this is worded sounds like it's something the driver does for me, but my understanding is that it's now something I have to do 

Suggestion:

```suggestion
- Required explicit dependency on the ``org.bson.codecs.record``
  module if your application deploys the driver in an OSGi container and
  relies on the driver for encoding and decoding Java records.
```

Reply:
changed to reflect that user must do task